README.md

```diff
@@ -49,6 +49,9 @@ Options to adjust generated callgrind data:
 - `-i` dump instructions, only possible with detail level "source"
 - `-m mode` default _mode_ is "callgraph" if detail level is not "object"
 
+Note: To collect with hardware counters you may have to do execute  
+`sh -c 'echo -1 >/proc/sys/kernel/perf_event_paranoid'`
```

Comment:
```suggestion
 `echo -1 >/proc/sys/kernel/perf_event_paranoid` as root.
````

Reply:
that doesn't work via `sudo`; what about using `sudo sh -c 'echo -1 >/proc/sys/kernel/perf_event_paranoid'` instead?`[and drop the duplicate verb "do" above, of course]

Reply:
We can't know whether the system has `su`, `sudo`, or both. We provide info on what needs to be done and with which permissions. It is up to user how to achieve that: `su`, `sudo`, `kdesu`, root login on console, ...

Reply:
you may even get to a system where you have that as read-only filesystem and must set it different. I'm going with

> Note: To collect with hardware counters you may have to adjust the kernel parameter `perf_event_paranoid` as root.

And as I'm already in the readme, I'll add the initial version of the "current limitation" and feature/difference text. Feel free to adjust it after the PR at any time.